mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxCounterDAOTest.java

```diff
@@ -79,6 +87,34 @@ public void incrementUnseenShouldAddOneWhenAbsent() throws Exception {
     }
 
     @Test
+    public void incrementUnseenShouldAddOneWhenAbsentOnMailboxCounters() throws Exception {
+        testee.incrementUnseen(MAILBOX_ID).join();
+
+        Optional<MailboxCounters> mailboxCounters = testee.retrieveMailboxCounters(mailbox).join();
+        assertThat(mailboxCounters.isPresent()).isTrue();
+        assertThat(mailboxCounters.get()).isEqualTo(new MailboxCounters(0L, 1L));
+    }
+
+    @Test
+    public void incrementCountShouldAddOneWhenAbsentOnMailboxCounters() throws Exception {
+        testee.incrementCount(MAILBOX_ID).join();
+
+        Optional<MailboxCounters> mailboxCounters = testee.retrieveMailboxCounters(mailbox).join();
+        assertThat(mailboxCounters.isPresent()).isTrue();
+        assertThat(mailboxCounters.get()).isEqualTo(new MailboxCounters(1L, 0L));
+    }
+
+    @Test
+    public void retrieveMailboxCounterShouldWorkWhenFullRow() throws Exception {
```

Comment:
Full row => both counter are incremented

Feel free to propose a commit for rename